src/ucp/core/ucp_request.h

```diff
@@ -204,7 +212,7 @@ struct ucp_request {
                 } rkey_ptr;
 
                 struct {
-                    ucs_ptr_map_key_t req_id;         /* the send request ID on receiver side */
+                    ucs_ptr_map_key_t sreq_id;        /* the send request ID on receiver side */
```

Comment:
should it be rreq_id rhen? see the comment

Reply:
better to name it as `remote_req_id`